.devcontainer/devcontainer.json

```diff
@@ -26,7 +26,14 @@
 		"source=${env:HOME}/.devcontainers/trino-bashhistory,target=/usr/local/share/bash_history,type=bind",
 		"source=${env:HOME}/scratch,target=${containerWorkspaceFolder}/scratch,type=bind",
 		"source=trino-dind-var-lib-docker,target=/var/lib/docker,type=volume",
-		"source=${env:HOME}/.m2,target=/usr/local/share/m2,type=bind"
+		"source=${env:HOME}/.m2,target=/usr/local/share/m2,type=bind",
+		"source=${env:HOME}/repositories/cccs-work/trino-azuread-plugin,target=${containerWorkspaceFolder}/trino-azuread-plugin,type=bind",
```

Comment:
I don't think we should be assuming where people clone their repos - we should definitely not hard-code these "cccs-work"  paths into the devcontainer. This will break for many people, including myself. The 'cccs-work' thing never made sense to me, since every repo I clone on this laptop is technically "CCCS work".

Reply:
Préfères-tu que j'inclus pas les projets et ça sera à la discrétion de chacun de le faire?

Reply:
Regarde ce que j'ai fait avec celui de DataHub. En gros, je présume que les devs se créent un "parent folder" où ils clonent tous leurs repos reliés à DH. Alors je mount le parent du répertoire qui contient le devcontainer.
https://github.com/CybercentreCanada/datahub/blob/7c19d052c36af90cf0cb50fcf5b22766cbb67359/.devcontainer/devcontainer.json#L27